src/esp/bindings_js/bindings.css

```diff
@@ -30,5 +30,5 @@
 }
 
 .hidden {
-  display: none;
+  display: none !important;
```

Comment:
Why is !important needed?

Reply:
Magnum somehow forces `style="display: block"` on the pre element.